### PR TITLE
Add rocketnoodles v0.1 to RLBotPack

### PR DESCRIPTION
**Pull request by GnomeBlue**
Add Rocketnoodles bot v0.1 to the RLBotPack.

### CODE REVIEW

File: RLBotPack/Rocketnoodles/src/scenario/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from scenario.default_scenario import DefaultScenario
+from scenario.shoot_ball_at_goal import ShootBallAtGoal
+from scenario.dribble_easy import DribbleEasy
+from scenario.dribble_hard import DribbleHard
+from scenario.intercept import Intercept
+from scenario.side_shot import SideShot
+from scenario.ball_through_center import BallThroughCenter
+
+__all__ = ["DefaultScenario", "ShootBallAtGoal", "DribbleEasy", "DribbleHard", "SideShot", "Intercept",
+           "BallThroughCenter"]
```
Comment (tarehart) [inline RLBotPack/Rocketnoodles/src/scenario/__init__.py:10]: You could omit your tests also, but up to you.

**Review by tarehart — COMMENTED** [review]
Very impressive! Looks good, just need to trim out a few things. Thanks for contributing this to the bot pack, and I hope you'll join a tournament soon 👍

**Comment by GnomeBlue** [comment]
@tarehart Thanks for the review, we'll have a look and we'll commit the changes soon!

File: RLBotPack/Rocketnoodles/logger/logger.cfg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Locations]
+# Path to loadout config. Can use relative path from here.
+looks_config = ./appearance.cfg
+
+# Path to python file. Can use relative path from here.
+python_file = ./logger.py
+
+# Name of the bot in-game
+name = logger
```
[the remark this answers is not in the document]
Comment (GnomeBlue) [reply RLBotPack/Rocketnoodles/logger/logger.cfg]: Removed the logger bot.

**Comment by GnomeBlue** [comment]
@tarehart I incorporated your feedback. I'll leave it to you to resolve the conversations once you checked them.

File: RLBotPack/Rocketnoodles/RefreshEnv.cmd [hunk 3]
```diff
@@ -0,0 +1,66 @@
+@echo off
+:: This file is taken from chocolatey:
+:: https://github.com/chocolatey/choco/blob/master/src/chocolatey.resources/redirects/RefreshEnv.cmd
```
[the remark this answers is not in the document]
Comment (GnomeBlue) [reply RLBotPack/Rocketnoodles/RefreshEnv.cmd]: Removed them and put them in the .gitignore.

File: RLBotPack/Rocketnoodles/src/scenario/__init__.py (hunk 1 above)
Comment (GnomeBlue) [reply RLBotPack/Rocketnoodles/src/scenario/__init__.py:10]: We would like to leave these in. They will probably play a bigger role in the future!

Comment (tarehart) [reply RLBotPack/Rocketnoodles/src/scenario/__init__.py:10]: OK!

**Review by tarehart — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**